### DISCUSSION TITLE
How to set up using two router ports, LAN and WAN

### DISCUSSION THREAD

**Discussion by ghost** [Ysurac/openmptcprouter#2003] [Q&A]
An attractive configuration might use one LAN port, and a different port connected to a switch, which also has the modems. It is possible to enter all the values in the wizard but the attempt fails. The error message says that the physical port is duplicated. That is true, I set every modem to use the same physical port which is the one connected to the switch. This seems very close to the approach that would be used with a Raspberry PI, but the tutorials that I am finding seem to have been made with a different version of the wizard, which is not asking for the physical port. If it is a supported configuration, how would it be set up ?

Also, if the config is practical, would there be any reason or need to have the modems all be in different subnets. Or to disable DHCP ? ( assuming DHCP clients are all on the LAN side )

**Comment by Ysurac** [comment]
If you want to use one physical port, you need to use MacVLAN.
DHCP need to be disabled for WAN, because multiples DHCP will make a conflict.
